### ISSUE TITLE
maybe beta 0.15 packages installation and glob bundling is broken

### ISSUE THREAD

**Issue by devmondo** [jspm/jspm-cli#617]
i installed jspm 0.15 beta, and used latest Aurelia skeleton, but jspm does not install all dependencies of dependencies, it just installs top ones

here is aurelia skeleton
https://github.com/aurelia/skeleton-navigation

as usual, just run jspm install , npm install then gulp watch, and you will see browser complaining about some dependencies
to fix the errors i have to add those sub dependencies or whatever you call them as top mapping.

also it seems that  gobbing bundling is not working ? 

i do 
```
jspm bundle src/**/* bundled.js
```

also bundle-sfx does not include system.js as mentioned in the docs to create a standalone bundle
so with this still browser complaining about system.js not available .
```
jspm bundle-sfx src/**/* bundle.js 
```

i hope i explained it well, thanks in advanced.

**Comment by guybedford** [comment]
Thanks for posting.

I'm not sure about the Aurelia install issue, will take a look.

Glob bundling requires wrapping in quotes to avoid population by the command line itself - `jspm bundle 'src/**/*' bundled.js`.

Yes bundle-sfx not including everything is a known bug, which was recently patched. A second beta will be coming out soon with the patch.

**Comment by devmondo** [comment]
@guybedford 

thanks for reply, and fast support, and by the way wish you the very best in your life, we know what hard times you have been through, our prayers go to you :)

**Comment by guybedford** [comment]
These issues should all be fixed in 0.15.0. Thanks for the assistance!
